Marlin/language_de.h

```diff
@@ -36,15 +36,20 @@
 #define WELCOME_MSG                         MACHINE_NAME " bereit"
 #define MSG_SD_INSERTED                     "SD-Karte erkannt"
 #define MSG_SD_REMOVED                      "SD-Karte entfernt"
+#define MSG_LCD_ENDSTOPS                    "Endstopp" // Max length 8 characters
 #define MSG_MAIN                            "Hauptmenü"
 #define MSG_AUTOSTART                       "Autostart"
-#define MSG_DISABLE_STEPPERS                "Motoren aus" // M84
+#define MSG_DISABLE_STEPPERS                "Motoren deaktivieren" // M84
 #define MSG_AUTO_HOME                       "Home" // G28
+#define MSG_AUTO_HOME_X                     "Home X"
+#define MSG_AUTO_HOME_Y                     "Home Y"
+#define MSG_AUTO_HOME_Z                     "Home Z"
 #define MSG_LEVEL_BED_HOMING                "Homing XYZ"
 #define MSG_LEVEL_BED_WAITING               "Klick für Start"
+#define MSG_LEVEL_BED_NEXT_POINT            "Nächste Koordinate"
 #define MSG_LEVEL_BED_DONE                  "Fertig"
 #define MSG_LEVEL_BED_CANCEL                "Abbruch"
-#define MSG_SET_HOME_OFFSETS                "Setze Homeoffsets"
+#define MSG_SET_HOME_OFFSETS                "Setze Homeversatz"
```

Comment:
I would leaf it now to "versatz" if we also want to change the rest. This file will be changed in the future often, so it makes no different to change it now back and in a lot of days again ;)

So it is equal, But now travis says that merge is ok and it don't need to be run again for one word.